leases/services/invoice.py

```diff
@@ -308,7 +308,7 @@ def resend_failed_invoices(self, profiles: dict) -> None:
 
         for order in orders:
             order.set_status(
-                OrderStatus.WAITING,
+                OrderStatus.OFFERED,
```

Comment:
The lease has to be in OFFERED status for the resending though `payments/utils.py:431` and `payments/schema/mutations.py:673`:
```
    if order.lease.status != LeaseStatus.OFFERED:
        raise ValueError(_("Cannot resend an order for a lease not in status OFFERED"))
```
and setting the order status to OFFERED also updates the lease status so perhaps it is best to set it to OFFERED here and in `payments.utils.py`

Reply:
Hmmmm good point. I think it would be redundant to set them to `OFFERED` here and then later on the same process mark it as `OFFERED` again. It makes sense that you can't re-send something that hasn't been sent.

Thinking...

Reply:
If we mark the status as OFFERED here, we don't have to do the status change in `resend_offer`.

Reply:
Fair enough, it would make sense that we should not move the status on resend, we should only resend `offered` offers, so it wouldn't make sense. Let's go with setting them as `OFFERED` 👍